### PR TITLE
Changes required to accommodate new Employee Mappings table

### PR DESCRIPTION
**Pull request by ashwin1111**

### CODE REVIEW

**Comment by ashwin1111** [comment]
Pylint will fail because fyle accounting mappings isn't released yet

File: requirements.txt [hunk 1]
```diff
@@ -9,7 +9,8 @@ django-q==1.3.3
 django-cors-headers==3.2.0
 django-picklefield==2.0
 django-request-logging==0.7.1
-fyle_accounting_mappings==1.4.1
+# fyle_accounting_mappings==1.5.0
+git+git://github.com/fylein/fyle-accounting-mappings@auto-map-emp-new-table
```
Comment (ashwin1111) [inline requirements.txt:13]: todo todo

File: apps/mappings/tasks.py [hunk 2]
```diff
@@ -452,7 +452,7 @@ def async_auto_map_employees(workspace_id: int):
     else:
         netsuite_connection.sync_vendors()
 
-    Mapping.auto_map_employees(destination_type, employee_mapping_preference, workspace_id)
+    AutoMapEmployees(workspace_id, destination_type, employee_mapping_preference).reimburse_mapping()
```
Comment (Shwetabhk) [inline apps/mappings/tasks.py:455]: why is this camel case?

File: apps/mappings/tasks.py [hunk 3]
```diff
@@ -484,7 +484,7 @@ def async_auto_map_ccc_account(workspace_id: int):
     fyle_connection = FyleConnector(refresh_token=fyle_credentials.refresh_token, workspace_id=workspace_id)
     fyle_connection.sync_employees()
 
-    Mapping.auto_map_ccc_employees('CREDIT_CARD_ACCOUNT', default_ccc_account_id, workspace_id)
+    AutoMapEmployees(workspace_id, 'CREDIT_CARD_ACCOUNT').ccc_mapping(default_ccc_account_id)
```
Comment (Shwetabhk) [inline apps/mappings/tasks.py:487]: why came case?

**Pull request merged**